### PR TITLE
Category model

### LINKED ISSUE #5: Create Category Model
Create Category Model

> Ori Granot

### PR DESCRIPTION
**Pull request by origranot**
Added Category model
#5

### CODE REVIEW

File: DGN/Models/Category.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    public class Category
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [DisplayName("Category name")]
+        [Required]
+        public string CategoryName { get; set; }
+
+        [DisplayName("The articles that using this category")]
```
Comment (bsuro10) [inline DGN/Models/Category.cs]: Should be: "Articles related to this category"

File: DGN/Models/Category.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        public IList<Atricle> ArticlesCategory { get; set; }
```
Comment (bsuro10) [inline DGN/Models/Category.cs]: The name of the property should be just "Articles"

**Review by bsuro10 — CHANGES_REQUESTED** [review]

File: DGN/Models/Category.cs (hunk 1 above)
Comment (origranot) [reply DGN/Models/Category.cs]: ok bro

File: DGN/Models/Category.cs (hunk 2 above)
Comment (origranot) [reply DGN/Models/Category.cs]: yes sir!

**Review by barBen — APPROVED** [review]
Approve

**Review by bsuro10 — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
please remove the unneeded using

**Pull request merged**